ui/cryptomaterial/dropdown.go

```diff
@@ -152,12 +156,18 @@ func (d *DropDown) layoutActiveIcon(gtx layout.Context, index int) D {
 }
 
 func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
-	item := d.items[itemIndex]
-	radius := Radius(0)
-	clickable := item.clickable
-	if !d.isOpen {
-		radius = Radius(8)
-		clickable = d.clickable
+	var item DropDownItem
+	if itemIndex > -1 {
+		item = d.items[itemIndex]
+	}
+
+	radius := Radius(8)
+	clickable := d.clickable
+	if d.isOpen {
+		radius = Radius(0)
+		if item.clickable != nil {
+			clickable = item.clickable
+		}
```

Comment:
`item.Clickable` is never nil when the dropdown is opened because the item at index 0 is always selected by default. Now that we are removing the ability to set a default selected index, by setting it to -1, when dropdown is opened, `item.clickable` will be nil, which is buggy and will cause the app to crash.